lib/commit-lint/last-commit-checker.js

```diff
@@ -10,6 +12,25 @@ module.exports = {
             emoji.emojify(':sparkles: Checking latest commit header format')
         );
 
-        return of(commitHasCorrectFormat(shell.exec('git log -1 --oneline')));
+        /**
+         * As we get an object after having executed the below `exec` statement in the CI env, we
+         * need to get it's string representation and replace the line breaks before testing the
+         * commit message against the Regexp
+         */
+        const lastCommit = shell
+            .exec('git log -1 --oneline')
+            .toString()
+            .replace('\n', '');
```

Comment:
```suggestion
        const lastCommit = shell.exec('git log -1 --oneline').stdout.replace('\n', '');
```
The shell-object have an attribute called `stdout`which contains the output as `String`-type.